feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/microprofile/opentracing-smallrye/main/module.xml

```diff
@@ -25,6 +25,11 @@
     </resources>
 
     <dependencies>
+        <module name="org.jboss.logging"/>
+        <module name="org.jboss.as.controller"/>
```

Comment:
See my comment about WildFlyTraceFactory.CONFIGURATIONS using simple String keys, which would remove the need for RuntimeCapability in this module, which removes the need for this dependency.

You need to add org.jboss.dmr though.